volatility/framework/plugins/windows/bigpools.py

```diff
@@ -0,0 +1,138 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
We should check with @awalters whether we need to update this header to 2020 for new files?